docs/versioning.md

```diff
@@ -0,0 +1,42 @@
+# Kubernetes API and Release Versioning
+
+Legend:
+
+* **Kube &lt;major&gt;.&lt;minor&gt;.&lt;patch&gt;** refers to the version of Kubernetes that is released. This versions all components: apiserver, kubelet, kubectl, etc.
+* **API vX[betaX]** refers to the version of the HTTP API.
+
+## Release Timeline
+
+### Minor version timeline
+
+* Kube 1.0
+* Kube 1.0.x: We create a 1.0-patch branch and backport critical bugs and security issues to it. Patch releases occur as needed.
+* Kube 1.1-alpha1: Cut from HEAD, smoke tested and released two weeks after Kube 1.0's release. Every two weeks a new alpha is released from HEAD.
```

Comment:
1.1-alpha1 doesn't fit the <major>.<minor>.<patch> scheme defined in the legend. Should it be something like 1.1-alpha1.0 so that we can patch the alpha releases if need be between the bi-weekly cuts?